### DISCUSSION TITLE
i3bar throwing an error in my config file.

### DISCUSSION THREAD

**Discussion by silvanathecat** [i3/i3#5698] [Q&A] [answered]
As the title says, i3bar is showing an error. It also isn't displaying.

This worked last night, and the error it's showing is that the bar block isn't a valid operator.
[error messages, image](https://github.com/i3/i3/assets/66652216/c9703590-bd88-465d-abdc-b519df71efd7)
[config file, txt](https://github.com/i3/i3/files/12750943/config.txt)

**Comment by okraits** [comment, marked as answer]
The closing bracket of the `mode "resize"` block is missing.

**Reply by silvanathecat** [reply]
Thank you so much!
